backport-changelog/6.7/7125.md

```diff
@@ -2,3 +2,4 @@ https://github.com/WordPress/wordpress-develop/pull/7125
 
 * https://github.com/WordPress/gutenberg/pull/61577
 * https://github.com/WordPress/gutenberg/pull/64610
+* https://github.com/WordPress/gutenberg/pull/65187
```

Comment:
Since these changes are going to [land first in core,](https://github.com/WordPress/wordpress-develop/pull/7125) I don't think we need this entry to mark this PR for backport, do we?

Reply:
I think we need it. Otherwise the `Check for a Core backport changelog entry` job is failing.

Reply:
Thanks, I wasn't sure either. I reverted the change and added the `Backport from WordPress Core` label so the job passes.